### DISCUSSION TITLE
Handling .gitignore

### DISCUSSION THREAD

**Discussion by paulo-ferraz-oliveira** [inaka/elvis_core#315] [General]
Would it be too "outside of scope" for `elvis_core` to deal with `.gitignore`? There's a couple of "good practices" already I can think of.

**Comment by elbrujohalcon** [comment]
I would approve it. Maybe not as a default ruleset but an optional one.

**Reply by paulo-ferraz-oliveira** [reply]
Yeah, same. An opinionated but optional rule 😄 (and configurable, too; some might - like I've also done - share an `elvis.config` file between repo.s)

**Comment by paulo-ferraz-oliveira** [comment]
https://github.com/inaka/elvis_core/issues/316
